api_v3/services/BaseEntryService.php

```diff
@@ -858,26 +858,32 @@ function indexAction($id, $shouldUpdate = true)
 	 * 
 	 * @action clone
 	 * @param string $entryId Id of entry to clone
+	 * @param KalturaBaseEntryCloneOptionsArray $cloneOptions
 	 * @param KalturaBaseEntry $updateEntry [optional] Attributes from these entry will be updated into the cloned entry
 	 * @return KalturaBaseEntry The cloned entry
 	 * @throws KalturaErrors::ENTRY_ID_NOT_FOUND
 	 */
-	function cloneAction( $entryId )
+	public function cloneAction( $entryId, $cloneOptions)
```

Comment:
$cloneOptions must have a null default value in order to avoid compatibility break.